index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-A3rJD856KowSb7dwlZdYEkO39Gagi7vIsF0jrRAoQmDKKtQBHUuLZ9AsSv4jD4Xa"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+
+    <title>Front End Developers</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="main">
+        <div class="esc-btn">
+          <button class="esc">✕</button>
+        </div>
+        <div class="header">
+          <h6>Add members to Front-end development team</h6>
+        </div>
+        <div class="form">
+          <form action="https://formspree.io/f/xlevenya" method="post">
+            <div class="search-box">
+              <button type="submit" class="search-btn btn" id="search-btn">
+                <i class="fa fa-search"></i>
+              </button>
+              <input
+                type="text"
+                name="search-word"
+                class="search"
+                placeholder="Find members"
+                required
+                id="search-word"
+              />
+            </div>
+            <div class="names-list">
+              <ul class="list-group" id="developers"></ul>
+            </div>
+            <div></div>
+            <div class="form-footer">
+              <button class="cancel-btn">Cancel</button>
+              <button class="save-btn" type="submit">save</button>
+            </div>
+          </form>
+        </div>
+      </div>
+    </div>
+  </body>
+  <script src="script.js"></script>
```

Comment:
Nice that you calling it after the body (To not stop rendering). This is the old fashioned way. Today you can use `async` and `defer` to fetch it without blocking

Reply:
thanks for this note , this is question of FE interview by the way .

Reply:
I used defer but didn't work correctly :(

Reply:
What is the issue?